packages/commerce-sdk-react/README.md

```diff
@@ -1,24 +1,51 @@
 # Commerce SDK React
 
-A library of React hooks for fetching data from Salesforce B2C Commerce.
+<p align="center">
```

Comment:
Non-Comment: Centre alignment is an interesting choice

Reply:
I feel like it's more eye catchy